### ISSUE TITLE
Ablation experiments of masking strategies

### ISSUE THREAD

**Issue by shmh40** [ecmwf/WeatherGenerator#491]
A small number of new masking strategies have been implemented for pre-training. These strategies have not been subjected to extensive experiment to determine their usefulness, and optimal hyperparameters (e.g. masking rate, healpix level choice) for each strategy. How exactly the usefulness of these strategies should be evaluated also needs to be determined.

These experiments will be carried out and model performance reported.

Experiment design and tracking document: https://gitlab.jsc.fz-juelich.de/hedgedoc/RSUt5YeCSR2zbjMQyR42eg?both

**Comment by shmh40** [comment]
@kctezcan can you assign yourself to this issue please? Thanks!

**Comment by kctezcan** [comment]
assign me
